frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts

```diff
@@ -0,0 +1,198 @@
+import { safeLoadAll } from 'js-yaml';
+import { checkErrors } from '../../support';
+import { isLocalDevEnvironment } from '../../views/common';
+import { detailsPage } from '../../views/details-page';
+import { infoMessage, hintMessage, warningMessage, warningToast } from '../../views/form';
+import { listPage } from '../../views/list-page';
+import { modal } from '../../views/modal';
+import { nav } from '../../views/nav';
+
+const PLUGIN_NAME = 'console-demo-plugin';
+const PLUGIN_PATH = '../../../dynamic-demo-plugin';
+const CHECK_UPDATE_WAIT = 30000;
+const PLUGIN_PULL_SPEC = Cypress.env('PLUGIN_PULL_SPEC');
```

Comment:
Should this be `CYPRESS_PLUGIN_PULL_SPEC` ?

Reply:
So the cypress.env api is as follows: https://docs.cypress.io/guides/guides/environment-variables#Option-3-CYPRESS_

I think we are ok with this definition.